gtsfm/common/image.py

```diff
@@ -34,12 +41,51 @@ def width(self) -> int:
         """
         return self.value_array.shape[1]
 
+    def __read_from_exif_image_width(self) -> float:
```

Comment:
nit: would a name like __derive_sensor_width_from_exif_image_width() capture the functionality more clearly? @akshay-krishnan what do you think? @ayushbaid

Reply:
yeah sounds good, or maybe `__read_sensor_width_from_exif`.

Reply:
Marking as resolved, thanks!